sdks/python/apache_beam/runners/dataflow/test_dataflow_runner.py

```diff
@@ -37,10 +38,13 @@ def run_pipeline(self, pipeline):
     """Execute test pipeline and verify test matcher"""
     options = pipeline._options.view_as(TestOptions)
     on_success_matcher = options.on_success_matcher
+    wait_duration = options.wait_until_finish_duration
+    is_streaming = options.view_as(StandardOptions).streaming
 
     # [BEAM-1889] Do not send this to remote workers also, there is no need to
     # send this option to remote executors.
     options.on_success_matcher = None
+    options.wait_until_finish_duration = None
```

Comment:
Do you need this here? I do not think this this is necessary based on BEAM-1889?

Reply:
The reason I add it here because the comment in line 45. If it's okay to send this option to remote executors, I prefer to remove it to keep code simple.

Reply:
It is ok to send it. The problem with `on_success_matcher` was it contains an `=` character and we found out that the workers had problem with parsing it out of the options.

Reply:
understand, will remove it.